client/src/App.js

```diff
@@ -40,9 +39,14 @@ function App() {
             <Events />
           </Route>
           {process.env.REACT_APP_FEATURE_REGISTRATION === 'true' && (
-            <Route path="/register">
-              <Register />
-            </Route>
+            <>
+              <Route path="/register">
+                <Register />
+              </Route>
+              <AuthProtectedRoute path="/setup">
```

Comment:
AuthProtectedRoute is used for routes that should only be accessible when logged in.